docs/source/markdown/podman-build.1.md

```diff
@@ -353,11 +365,15 @@ unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
 **--net**, **--network**=*string*
 
 Sets the configuration for network namespaces when handling `RUN` instructions.
-The configured value can be "" (the empty string) or "container" to indicate
-that a new network namespace should be created, or it can be "host" to indicate
-that the network namespace in which `podman` itself is being run should be
-reused, or it can be the path to a network namespace which is already in use by
-another process.
+
+Valid _mode_ values are:
+
+Valid _mode_ values are:
+
+- **none**: no networking;
+- **host**: use the Podman host network stack. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure;
+- **ns:**_path_: path to a network namespace to join;
+- `private`: create a new namespace for the container (default)
```

Comment:
Just private?  I don't recall, but thought there were one or two more....

Reply:
Well there is private, none, host and ns.  There is also container: for podman run, but this does not work with build.